### PR TITLE
Update routes to include all paths

### PR DESCRIPTION
**Pull request by catb0591**
Goal of this PR is to finalize all the routes needed for our application

Created a Text Campaign route that will handle the elements of our campaign. Added additional routes to handle deleting contacts, updating contacts, deleting texts, updating texts, displaying all texts, displaying a specific text. I updated naming convention for route files and controller files.

### CODE REVIEW

**Review by IkmosaO — APPROVED** [review]
All the routes look good.

File: index.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
-const textMessageRouter = require('./routes/textMessage.routes')
+const contactsRouter = require('./routes/contactsRoutes')
+const textMessageRouter = require('./routes/textMessageRoutes')
+const textCampaignRouter = require('./routes/textCampaignRoutes')
 
 const app = express()
 
-app.use('api/contact', contactsRouter)
-app.use('api/text-message', textMessageRouter)
+app.use('api/contacts', contactsRouter)
```
Comment (IgnacioOAF) [inline index.js]: nitpick: do we want to use `api/contact` instead of the plural so it matched the other routes?

**Review by IgnacioOAF — APPROVED** [review]
Nice! Left a nitpick comment.

File: index.js (hunk 1 above)
Comment (catb0591) [reply index.js]: All set! Added a commit correcting this so it is no longer plural

**Pull request merged**